website/assets/styles/layout.less

```diff
@@ -181,7 +181,7 @@ body.detected-mobile {
   [purpose='page-wrap'] {
     padding-bottom: 230px;
     [purpose='page-footer'] {
-      height: 230px;
+      height: 260px;
```

Comment:
@mike-j-thomas just making sure: has this been tested?  I would think the bottom padding would have to updated as well for this to work properly with scrolling.

Reply:
(@rachaelshaw says that you wouldn't have been able to tell the difference when testing except on a mostly blank page -- this is a special case based on how the sticky footer works. So think of it as a preventative measure)

Reply:
Gotcha, thanks. I just updated them to match.